operator/internal/manifests/openshift/opa_openshift.go

```diff
@@ -12,13 +12,15 @@ import (
 )
 
 const (
-	envRelatedImageOPA     = "RELATED_IMAGE_OPA"
-	defaultOPAImage        = "quay.io/observatorium/opa-openshift:latest"
-	opaContainerName       = "opa"
-	opaDefaultPackage      = "lokistack"
-	opaDefaultAPIGroup     = "loki.grafana.com"
-	opaMetricsPortName     = "opa-metrics"
-	opaDefaultLabelMatcher = "kubernetes_namespace_name"
+	envRelatedImageOPA = "RELATED_IMAGE_OPA"
+	defaultOPAImage    = "quay.io/observatorium/opa-openshift:latest"
+	opaContainerName   = "opa"
+	opaDefaultPackage  = "lokistack"
+	opaDefaultAPIGroup = "loki.grafana.com"
+	opaMetricsPortName = "opa-metrics"
+	// TODO: should we make that more configurable rather than hardcoded? E.g. via lokistack or tenant config?
```

Comment:
@periklis what do you think about this comment? Maybe as a follow-up, it would be interesting to make these opa matchers more configurable & avoid hardcoding?

Reply:
Let's save this for later or upon user request